### DISCUSSION TITLE
Allow changing post statuses right from the calendar

### DISCUSSION THREAD

**Discussion by OSTraining** [publishpress/PublishPress-Planner#1261] [Feature Requests]
Right now we can only change that editing  the post.

#### Support ticket (private)
https://secure.helpscout.net/conversation/536932600/65181/

**Comment by stevejburge** [comment]
One more vote for this:

> It would be great to be able to set date and time for scheduling, as well as change the status to scheduled or even published from the calendar.

**Comment by stevejburge** [comment]
One more request for this:

> As the calendar is already handling the process of selecting the date, there should be a drop down to change the status of the post (just like in single post).
> 
> For now, we have to edit the post to change its status from draft to schedule for the posts that have already been reviewed by our editors.
